### PR TITLE
Adding Bootstrap button.js examples to buttons test page

### PR DESCRIPTION
**Pull request by rhamilto**
The Bootstrap button.js examples weren't part of the test page, so this adds them since we'll be promoting their use in the near future.

@andresgalante or @erundle, please review and merge.

### CODE REVIEW

File: tests-src/buttons.html [hunk 1]
```diff
@@ -484,4 +484,32 @@
             <li><a href="#">Separated link</a></li>
           </ul>
         </div><!-- /btn-group -->
-      </div><!-- /btn-toolbar -->
\ No newline at end of file
+      </div><!-- /btn-toolbar -->
+      <h2>buttons.js</h2>
```
Comment (erundle) [inline tests-src/buttons.html]: I know this is what bootstrap calls its example but can we find a better name for this? "Advanced Buttons - requires bootstrap.js"

**Comment by andresgalante** [comment]
I don't have any preference between "Buttons" or "Advanced Buttons". The rest :+1:

**Comment by rhamilto** [comment]
Changed heading to @erundle's suggestion.

**Comment by erundle** [comment]
Thanks. Will merge

**Pull request merged**